### ISSUE TITLE
RedisBungee Support

### ISSUE THREAD

**Issue by apatuka** [MultiChat/Development#88]
**Is your feature request related to a problem? Please describe.**
I'm about to connect BungeeCord servers with RedisBungee, and adding RedisBungee support can be useful since MultiChat is only plugin that connects chat servers with global chat. 

I'm planning to use [this](https://www.spigotmc.org/resources/proxychannels-redisbungee-support.34255/) but it's not automatic.

**Describe the solution you'd like**
Adding RedisBungee support to MultiChat will help to many networks that handles more than 1 BungeeCord server connected with Redis to keep all chats on sync.

**Describe alternatives you've considered**
ProxyChannels but it's not confortable using a prefix to send global chat.

**Additional context**
Thank you !

**Comment by OllieMartin** [comment]
Yes this is definitely on my list to consider for future updates! I cannot promise exactly when as there are a lot of features in the works, but I will keep it as near to the top as I can :)
